project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clj-zeppelin "0.1.2"
+(defproject clj-zeppelin "0.1.1"
```

Comment:
What's the version on clojars? If that's 0.1, then the repo should have 0.2-snapshot (the next snapshot version)